app/controllers/admin/routes.py

```diff
@@ -91,6 +93,8 @@ def createEvent(templateid, programid=None):
     # Try to save the form
     if request.method == "POST":
         try:
+            if fileDoesNotExist:
+                attachmentFiles = None
```

Comment:
Let's just do this up where I made the comment, and avoid having to do a conditional here at all